src/SGB_Command_Undocumented.md

```diff
@@ -4,16 +4,10 @@ The following information has been extracted from disassembling a SGB1v2
 firmware; it should be verified on other SGB revisions.
 
 The SGB firmware explicitly ignores all commands with ID >= $1E. This
-leaves undocumented commands $19 to $1D inclusive.
+leaves undocumented commands $1A to $1D inclusive.
 
 ## Stubbed commands
 
 Commands $1A to $1F (inclusive)'s handlers are stubs (only contain an
 `RTS`). This is interesting, since the command-processing function
 explicitly ignores commands $1E and $1F.
-
-## SGB command $19
-
-The game _Donkey Kong_ (1994) appears to send this command, and it appears
```

Comment:
Looks good to me! The only thing I would do is to keep a reference to Donkey Kong as an example of game that does it